staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/internal/testdata/swagger.json

```diff
@@ -3678,20 +3678,7 @@
     },
     "io.k8s.api.core.v1.WindowsSecurityContextOptions": {
       "description": "WindowsSecurityContextOptions contain Windows-specific options and credentials.",
-      "properties": {
```

Comment:
have you regenerated this file or edited manually? I think you need to fix types.go file and regenerate swagger `make update`

Reply:
Hi,
How can i do it? Could you give an exemple?